.buildkite/pipeline.yml

```diff
@@ -100,17 +89,6 @@ steps:
    timeout_in_minutes: 30
    agents:
      queue: "docker"
- - label: test rpm binary packages via docker
```

Comment:
Maybe instead of running them in PRs let's at least run them in master or on schedule? Master builds are rare and can be cancelled, and if they fail we will likely notice that.